src/features/tasks/api/task.handler.ts

```
@@ -41,6 +45,20 @@ export class TaskHandler {
         // Validate shape defensively using DTO
         const { TaskListResponseDto } = await import('@/features/tasks/dto/task.dto')
         const data = TaskListResponseDto.parse(result.data)
+        // Overdue notifications (best-effort, deduped)
+        try {
+          const { NotificationRepository } = await import('@/features/notifications/repo/notification.repository')
+          const repo = new NotificationRepository()
+          const now = Date.now()
+          for (const t of data.tasks as any[]) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)